src/Workspaces/Core/Portable/Rename/RenameSymbolAnnotation.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Rename
+{
+    /// <summary>
+    /// Provides a way to produce and consume annotations for rename
+    /// that contain the previous symbol serialized as a <see cref="SymbolKey" />.
+    /// This annotations is used by <see cref="Workspace.TryApplyChanges(Solution)" /> 
+    /// in some cases to notify the workspace host of refactorings.
+    /// </summary>
+    internal static class RenameSymbolAnnotation
+    {
+        public const string RenameSymbolKind = nameof(RenameSymbolAnnotation);
+
+        public static SyntaxAnnotation Create(ISymbol oldSymbol)
+        => new SyntaxAnnotation(RenameSymbolKind, SerializeData(oldSymbol));
```

Comment:
Indent.

Reply:
Why does our formatter not catch these :(